util/dataset.go

```diff
@@ -84,6 +85,10 @@ type Os struct {
 	Windows interface{} `config:"windows" json:"windows,omitempty" yaml:"windows,omitempty"`
 }
 
+type Elasticsearch struct {
+	IngestPipelineName string `config:"ingest_pipeline.name,omitempty" json:"ingest_pipeline.name,omitempty" yaml:"ingest_pipeline.name,omitempty"`
```

Comment:
@ycombinator Not 100% happy with the name yet. Any input?

Reply:
I think just `IngestPipeline` is enough? Even looking at Filebeat fileset's `manifest.yml`s, I don't think we've ever needed to define any sub-properties of ingest pipelines?

Reply:
There are 2 properties for ingest pipeline I had in mind:

* Path to the file locally
* Name defined in the manifest instead of generated name in the package manager

That is why I made it an object to leave this door open.